src/Services/PrescriptionService.php

```diff
@@ -78,70 +80,195 @@ public function getAll($search = array(), $isAndCondition = true, $puuidBind = n
+                        ,NULL as encounter
+                        ,users.id AS provider_id
+                        ,NULL as drug_uuid
+                        ,lists_medication.drug_dosage_instructions
+                        ,CASE 
+                                WHEN lists.enddate IS NOT NULL AND lists.activity = 1 THEN 'completed'
+                                WHEN lists.activity = 1 THEN 'active'
+                                ELSE 'stopped'
+                        END as 'status'
+                    FROM
+                        lists
+                    LEFT JOIN 
+                            users ON users.username = lists.user
+                    LEFT JOIN
+                        lists_medication ON lists_medication.list_id = lists.id
+                    WHERE
+                        type = 'medication'
+                ) combined_prescriptions
+                LEFT JOIN
+                (
+                  SELECT
+                    option_id AS route_id
+                    ,title AS route_title
+                    ,codes AS route_codes
+                  FROM list_options
+                  WHERE list_id='drug_route'
+                ) routes_list ON routes_list.route_id = combined_prescriptions.route
+                LEFT JOIN
+                (
+                  SELECT
+                    option_id AS interval_id
+                    ,title AS interval_title
+                    ,codes AS interval_codes
+                  FROM list_options
+                  WHERE list_id='drug_route'      
+                ) intervals_list ON intervals_list.interval_id = combined_prescriptions.interval
+                LEFT JOIN
+                (
+                  SELECT
+                    option_id AS unit_id
+                    ,title AS unit_title
+                    ,codes AS unit_codes
+                  FROM list_options
+                  WHERE list_id='drug_route'
+                ) units_list ON units_list.unit_id = combined_prescriptions.unit
+                LEFT JOIN (
+                    select uuid AS puuid
+                    ,pid
+                    FROM patient_data
+                ) patient
+                ON patient.pid = combined_prescriptions.patient_id
+                LEFT JOIN (
+                    SELECT
+                        encounter,
+                        uuid AS euuid
+                    FROM form_encounter
+                ) encounter
+                ON encounter.encounter = combined_prescriptions.encounter
+                LEFT JOIN (
+                    SELECT 
+                           id AS practitioner_id
+                           ,uuid AS pruuid
+                    FROM users
+                    WHERE users.npi IS NOT NULL AND users.npi != ''
+                ) practitioner
+                ON practitioner.practitioner_id = combined_prescriptions.provider_id";
```

Comment:
dang, now that is a sql query!

![2DV-2](https://user-images.githubusercontent.com/278968/128593776-df07960c-8721-4f94-93f4-1b1b18baf92a.gif)